Source/AtomicNET/NETNative/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ add_library(AtomicNETNative SHARED ${SOURCE_FILES} ${CSHARP_BINDINGS_SOURCE})
 if (ATOMIC_DESKTOP)
     add_dependencies(AtomicNETNative AtomicToolCheckScripts)
 endif ()
-target_link_libraries(AtomicNETNative AtomicApp AtomicNETScriptBindings AtomicNETScript AtomicJS AtomicPlayerLib AtomicPlayerJS Atomic)
+target_link_libraries(AtomicNETNative AtomicApp AtomicNETScriptBindings AtomicNETScript AtomicJS AtomicPlayerLib AtomicPlayerJS Atomic ToolCore Poco Iphlpapi)
 target_include_directories(AtomicNETNative PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
A great deal of work has gone into Atomic's modularity.  This isn't always the most expedient path, and sometimes in production it makes sense to connect wires.  I don't think that is the case here, you really want to keep ```AtomicNETNative.dll``` unlinked from ToolCore and associated dependencies, see review summary comment.